modules/aws-frontend/variables.tf

```diff
@@ -56,3 +56,8 @@ variable "content_security_policy" {
     "worker-src"  = "blob:"
   }
 }
+
+variable "web_acl_id" {
```

Comment:
Please run `./scripts/generate-docs.sh` to generate documentation out of Terraform code.